lib/proxy/plugin_validators.rb

```diff
@@ -8,7 +8,7 @@ def initialize(plugin, setting_name, params, predicate)
     end
 
     def required_setting?
-      @plugin.plugin_default_settings.has_key?(@setting_name)
+      @plugin.plugin_default_settings.has_key?(@setting_name) && !@plugin.plugin_default_settings[@setting_name].nil?
```

Comment:
Why do we need this change?

Reply:
Prior to this commit, not specifying a setting at all means that the setting is optional. For the setting to show up as an env variable, we need to explicitly define all settings. So setting the default to `nil` now means the setting is optional. Makes sense?